app/routers/todos.py

```
@@ -0,0 +1,108 @@
+
+
+
+router = APIRouter(prefix="/todos", tags=["Todos"])
+
+
+# Todoの一覧取得
+@router.get("/todos", response_model=list[schemas.TodoOut])
+def get_my_todos(
+        skip: int = 0,
+        limit: int = 10,
+        db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)